ArduCopter/mode.cpp

```diff
@@ -363,6 +356,12 @@ void Copter::exit_mode(Mode *&old_flightmode,
     }
 #endif
 
+#if MODE_ZIGZAG_ENABLED == ENABLED && SPRAYER_ENABLED == ENABLED
```

Comment:
This shouldn't be conditional upon the sprayer being enabled.

Reply:
Thank you for the review.
Fixed it.